### DISCUSSION TITLE
Swiper Js not working in custom WordPress theme

### DISCUSSION THREAD

**Discussion by kindnessrho** [nolimits4web/swiper#6500] [Q&A]
I have a html website that I have converted to a custom wp theme but swiper js does not work. It works well in the plain html website with out any issues but it doesn't work in the custom wp theme. I have linked my js and css files properly in the functions.php but still not working? How do I fix this

**Comment by shikkaba** [comment]
@kindnessrho I realize this is old, and if you found your answer that you would've found it ages ago, but to answer a question like this we would need more details. Wordpress is highly customizable, so someone would have to know what was put in functions.php and how your javascript was written.
